spec/index.html

```diff
@@ -443,32 +525,50 @@ <h3>Triples</h3>
 
     <ul>
       <li>the <dfn>subject</dfn>, which is an
-      <a>IRI</a> or a <a>blank node</a></li>
+      <a>IRI</a>, a <a>blank node</a>, or a <a>quoted triple</a></li>
 
       <li>the <dfn>predicate</dfn>, which is an <a>IRI</a></li>
 
       <li>the <dfn>object</dfn>, which is an <a>IRI</a>,
-      a <a>literal</a> or a <a>blank node</a></li>
+      a <a>literal</a>, a <a>blank node</a>, or a <a>quoted triple</a></li>
     </ul>
 
     <p>An RDF triple is conventionally written in the order subject,
       predicate, object.</p>
 
+    <p>RDF triple is usually written "<a>triple</a>".<p>
+
+    <p>A <dfn data-local-lt="quoted">quoted triple</dfn> is an <a>RDF term</a> representing a <a>triple</a>
+      used as the <a>subject</a> or <a>object</a> of another triple.</p>
+
+    <p>An <dfn data-local-lt="asserted">asserted triple</dfn>
+      is an <a>RDF triple</a> that is an element of an <a>RDF graph</a>.</p>
+
     <p>The set of <span id="dfn-nodes"><!-- obsolete term--></span><dfn data-lt="node">nodes</dfn> of an <a>RDF graph</a>
-      is the set of subjects and objects of triples in the graph.
+      is the set of <a>subjects</a> and <a>objects</a> of <a>triples</a> in the graph
+      plus all the <a>nodes</a> of those <a>subjects</a> and/or its <a>objects</a>,
+      recursively, for <a>quoted triples</a>.
       It is possible for a predicate IRI to also occur as a node in
       the same graph.</p>
 
-    <p><a>IRIs</a>, <a>literals</a> and
-      <a>blank nodes</a> are collectively known as
+    <p><a>IRIs</a>, <a>literals</a>,
+      <a>blank nodes</a>, and <a>quoted triples</a> are collectively known as
       <span id="dfn-rdf-terms"><!-- obsolete term--></span><dfn data-lt="rdf term">RDF terms</dfn>.</p>
 
-    <p><a>IRIs</a>, <a>literals</a>
-      and <a>blank nodes</a> are distinct and distinguishable.
+    <p><a>IRIs</a>, <a>literals</a>,
+      <a>blank nodes</a>, and <a>quoted triples</a> are distinct and distinguishable.
       For example, <code>http://example.org/</code> as a string literal
       is neither equal to <code>http://example.org/</code> as an IRI,
       nor to a blank node with the <a>blank node identifier</a>
       <code>http://example.org/</code>.</p>
+
+    <p class="note">In an <a>RDF graph</a>,
+      a <a>triple</a> may occur as both a <a>quoted</a> triple term and as an <a>asserted</a> triple.</p>
```

Comment:
```suggestion
      a <a>triple</a> may occur as either, neither, or both a <a>quoted</a>
      triple term and an <a>asserted</a> triple.</p>
```

Reply:
Hard to see how "neither" factors in here, as a triple which is neither quoted nor asserted is not in the graph at all, so isn't really a triple of discourse.

Reply:
Oh, you're right. I guess "either or both" covers my concern (which is the possible implication of "all or nothing"; i.e., "either" disallowed). I've revised my suggestion.

Reply:
I've edited mine to match your latest, @gkellogg. Either may be committed to the PR to make me happy.